src/app/modules/group/pages/group-composition/group-composition.component.html

```diff
@@ -2,11 +2,11 @@
   <alg-section icon="fa fa-users" label="Current Composition">
     <p>...</p>
   </alg-section>
-  <alg-group-join-by-code
-    [group]="group"
-    (refreshRequired)="refreshGroupInfo()"
-    *ngIf="canCurrentUserManageMembers"
-  ></alg-group-join-by-code>
+  <alg-group-join-by-code [group]="group" (refreshRequired)="refreshGroupInfo()" *ngIf="canCurrentUserManageMembers">
+  </alg-group-join-by-code>
+
+  <alg-group-invite-users [group]="group" (refreshRequired)="refreshGroupInfo()" *ngIf="canCurrentUserManageMembers">
```

Comment:
`refreshRequired` devrait correspondre à un "@Output" dans ton composant.

Reply:
Pour le moment, j'appelle `this.refreshRequired.emit()` dans la fonction `onInvite()`, donc quand on clique sur le bouton Invite